docs/pages/image-extractor.rst

```diff
@@ -100,114 +113,375 @@ there is no semantic data which is why the semantic output below is not represen
+Each instance of an ImageExtractor has a pose extractor (an instance of PoseExtractor). The pose extractor defines how camera poses are programmatically determined so that the image
+extractor knows how to manipulate the camera position and angle to extract an image from habitat
+sim. Uses can write their over subclass of PoseExtractor to define custom ways of getting these camera poses. All custom pose extractors must inherit from the PoseExtractor abstract class and
+override the extract_poses method. Further, the user must register the pose extractor using
+habitat_sim.registry (i.e. adding the @registry.register_pose_extractor(name) decorator). This allows you to pass the name of your custom pose extractor to the ImageExtractor constructor. For more detailed examples of using the habitat registry, see `this code`_.
 
-The extract_poses method takes three required arguments:
+`Overriding required methods`_
+------------------------------
 
-* labels: List[float]: A list of numeric labels of points we are looking for
-* view: numpy.ndarray: A 2 dimensional array representing the topdown view of a scene.
-* fp: str: The filepath of the scene.
+You should not override the init method in your custom pose extractor, but if you need to you must take in two required arguments, which are the topdownview(s) and the pixels_per_meter, and pass them to the constructor of PoseExtractor superclass using ``super().__init__(topdownviews, pixels_per_meter)``. This is because the ImageExtractor will pass your pose extractor these arguments which the PoseExtractor superclass will use to convert between the coordinate system of those topdown views and the scene coordinate system.
 
-The job of the extract_poses method is to return a list of poses where each pose is a four-tuple
-of (<camera position>, <point of interest>, <label>, <scene filepath>). The camera position is the
+You must override the extract_poses method which takes two required arguments:
+
+* view (numpy.ndarray): A 2 dimensional array representing the topdown view of a scene. This array is used to determine valid camera positions. 1.0 in the array means the position is valid and 0.0 means the position is not valid.
+* fp (str): The filepath of the scene that the view represents.
+
+The job of the extract_poses method is to return a list of poses where each pose is a three-tuple
+of (<camera position>, <point of interest>, <scene filepath>). The camera position is the
 coordinates of the camera in the space of the topdown view. The point of interest is the position where
-the camera will be pointing in the space of the topdown view. The label and filepath are the class
-label of what the camera is pointing at and the scene filepath the pose comes from respectively. Below
+the camera will be pointing in the space of the topdown view. The filepath is the scene filepath the pose comes from, which is necessary when the ImageExtractor is querying images from multiple different scenes. Below
```

Comment:
Can we explain these concepts with pictures? Top-down maps, example camera placements, etc. 

Also, repeating my comment from the beginning again -- it would be nice to understand first what's done by default, so I can know why/whether I'd want to change it.